libs/sdk-ui-kit/src/List/ListItem.tsx

```diff
@@ -25,90 +25,144 @@ export interface ISingleSelectListItemProps {
     hideDelayBubble?: number;
     isSelected?: boolean;
     isMenu?: boolean;
+    accessibilityConfig?: IMenuAccessibilityConfig;
+    tabIndex?: number;
+    elementType?: "div" | "button";
 
     onClick?: (e: React.MouseEvent<HTMLElement>) => void;
     onMouseOver?: (e: React.MouseEvent<HTMLElement>) => void;
     onMouseOut?: (e: React.MouseEvent<HTMLElement>) => void;
 }
+
 /**
  * @internal
  */
-export interface ISingleSelectListItemState {
-    isOverflowed: boolean;
-}
+const DivElement = forwardRef<HTMLDivElement, ISingleSelectListItemProps & { children?: React.ReactNode }>(
```

Comment:
Maybe call it `AccessibleDiv`? It takes `type` prop which is not native to `div` however but I would ignore that.

Below as well. `AccessibleButton`?

Reply:
It is not so much about accessibility as about two types of elements used for rendering list item.